### PR TITLE
Export logs

### LINKED ISSUE #491: Display chainguardian version
Display version in nav bar under about or something, maybe print version in bottom right corner and in logs.

### LINKED ISSUE #492: Menu - extract log file
there should be an option on the menu to open the logs folder for simplified navigating users to it

### PR DESCRIPTION
**Pull request by BeroBurny**
closes #492 
closes #491

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -180,6 +180,8 @@
     "@fortawesome/react-fontawesome": "^0.1.12",
     "@reduxjs/toolkit": "^1.4.0",
     "@sentry/electron": "2.0.3",
+    "@types/archiver": "^5.1.0",
```
Comment (mpetrunic) [inline package.json:183]: dev dependency

**Comment by BeroBurny** [comment]
additionally added #491

File: src/main/menu/menu.ts [hunk 2]
```diff
@@ -26,7 +30,26 @@ const template = [
                     await shell.openExternal("https://discord.gg/4GBwH52cFb");
                 },
             },
-            ...(isMac ? [{role: "about"}] : []),
+            ...(isMac
```
Comment (mpetrunic) [inline src/main/menu/menu.ts:33]: what's different on mac?

Comment (BeroBurny) [reply src/main/menu/menu.ts:33]: mac have own native about
image
while others
Screenshot from 2021-02-11 13-09-50
 solutions need own

**Review by mpetrunic — APPROVED** [review]

**Pull request merged**